src/contracts/core/RewardsCoordinatorStorage.sol

```diff
@@ -57,7 +57,7 @@ abstract contract RewardsCoordinatorStorage is IRewardsCoordinator {
      */
     DistributionRoot[] internal _distributionRoots;
 
-    /// Slot 3
+    /// Slot 2
```

Comment:
Updated the comment here. It is in fact Slot 2 since we start from Slot 0 (rather than Slot 1)